gamestonk_terminal/cryptocurrency/overview/overview_controller.py

```diff
@@ -131,6 +136,9 @@ def __init__(self, queue: List[str] = None):
             choices["cpcontracts"]["-s"] = {
                 c: None for c in coinpaprika_model.CONTRACTS_FILTERS
             }
+            choices["hm"] = {c: None for c in get_categories_keys()}
+            choices["hm"]["--category"] = {c: None for c in get_categories_keys()}
```

Comment:
didn't we decide to just have the main one, e.g. `hm something` on autocomplete. To not add delay to all commands? Or did I just misremembered that?

Reply:
my bad. forgot about this